README.md

```diff
@@ -1 +1,188 @@
-# live-demo
\ No newline at end of file
+<a name="readme-top"></a>
+
+<!--
+HOW TO USE:
+This is an example of how you may give instructions on setting up your project locally.
+
+Modify this file to match your project and remove sections that don't apply.
+
+REQUIRED SECTIONS:
+- Table of Contents
+- About the Project
+  - Built With
+  - Live Demo
+- Getting Started
+- Authors
+- Future Features
+- Contributing
+- Show your support
+- Acknowledgements
+- License
+
+OPTIONAL SECTIONS:
+- FAQ
+
+After you're finished please remove all the comments and instructions!
+-->
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Portfolio] <a name="about-project"></a>
+
+**[Portfolio]** is a webpage build to display my projects, about me, skill and how to get in touch with me.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+> Describe the tech stack and include only the relevant sections that apply to your project.
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li>Html</li>
+    <li>Css</li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+> Describe between 1-3 key features of the application.
+
+- **[Recent and past projects]**
+- **[Skills]**
+- **[Contact info]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- A web browser e.g Chrome
+- A code editor e.g VS code
+- A terminal
+- Git
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  cd my-folder
+  git clone https://github.com/citec-47/portfolio.git
+```
+
+### Install
+
+Install this project with:
+
+```sh
+  cd my-project
+ npm install
+```
+
+### Deployment
+
+Still under construction:
```

Comment:
- [ ] Kindly consider editing this part in your readme as now you have deployed your app and this will make your readme more professional it will avoid confusion as well 
- you can do so by adding this [Live Demo](https://citec-47.github.io/live-demo/) under this title
and this will help you to fulfill any of these requirements
> - Update the README of your repository to include a link to the online version.
> - If by any chance you have deployed your app earlier, open an issue with a link to the online version of your app. Otherwise, submit a link to the PR with changes required for the deploy.